examples/arkode/CXX_superludist/ark_brusselator1D_FEM_sludist.out

```diff
@@ -118,5 +118,5 @@ Final Solver Statistics:
    Total linear solver setups = 31
    Total number of Jacobian evaluations = 11
    Total number of Newton iterations = 1127
-   Total number of nonlinear solver convergence failures = 0
+   Total number of nonlinear solver convergence failures = 10
```

Comment:
Do we have any idea why this jumped from 0 failures to 10?  Were we just not capturing failures that occurred, or did convergence deteriorate due to these changes?

Reply:
I suspect this is due to fixing how we count nonlinear solver convergence failures (before a failure with a stale Jacobian was not counted). This file was not updated with that change because of a bug in the CI that caused the SuperLU_DIST tests not to run (fix in this PR). We could double check this with `ARKStepGetNumStepSolveFails` which should return the old failure count.

Reply:
Looks like there are 11 Jacobian evaluations so 10 NLS convergence failures should be correct.